ui/app/AppLayouts/Chat/ChatColumn/SuggestionBox.qml

```diff
@@ -108,42 +115,60 @@ Rectangle {
         signal suggestionClicked(var item)
 
-        delegate: Rectangle {
-            id: rectangle
-            color: listView.currentIndex === index ? Style.current.backgroundHover : Style.current.transparent
-            border.width: 0
-            width: parent.width
-            height: 42
-            radius: Style.current.radius
-
-            StatusImageIdenticon {
-                id: accountImage
-                width: 32
-                height: 32
-                anchors.verticalCenter: parent.verticalCenter
-                anchors.left: parent.left
-                anchors.leftMargin: Style.current.smallPadding
-                source: model.identicon
-            }
+        DelegateModelGeneralized {
+            id: mentionsListDelegate
+            lessThan: [
+                function(left, right) {
+                    if (!formattedPlainTextFilter) {
+                        return true
+                    }
 
-            StyledText {
-                text: model[container.property.split(",").map(p => p.trim()).find(p => !!model[p])]
-                color: Style.current.textColor
-                anchors.verticalCenter: parent.verticalCenter
-                anchors.left: accountImage.right
-                anchors.leftMargin: Style.current.smallPadding
-                font.pixelSize: 15
-            }
+                    const leftMatches = left[container.property.find(p => !!left[p])].toLowerCase().startsWith(formattedPlainTextFilter)
+
+                    const rightMatches = right[container.property.find(p => !!right[p])].toLowerCase().startsWith(formattedPlainTextFilter)
 
-            MouseArea {
-                cursorShape: Qt.PointingHandCursor
-                anchors.fill: parent
-                hoverEnabled: true
-                onEntered: {
-                    listView.currentIndex = index
+                    return leftMatches && !rightMatches
                 }
-                onClicked: {
-                    container.itemSelected(model, filterItem.lastAtPosition, filterItem.cursorPosition)
+            ]
+
+            model: container.suggestionsModel
+
+            delegate: Rectangle {
+                color: listView.currentIndex === index ? Style.current.backgroundHover : Style.current.transparent
+                border.width: 0
+                width: parent.width
+                height: 42
+                radius: Style.current.radius
+
+                StatusImageIdenticon {
+                    id: accountImage
+                    width: 32
+                    height: 32
+                    anchors.verticalCenter: parent.verticalCenter
+                    anchors.left: parent.left
+                    anchors.leftMargin: Style.current.smallPadding
+                    source: model.identicon
+                }
+
+                StyledText {
+                    text: model[container.property.find(p => !!model[p])]
```

Comment:
TIL!

Reply:
Ah! it was already part of the suggestionfilter code, but nevertheless, I did not know it was possible to get a model property like this